TESTING.md

```diff
@@ -249,6 +249,54 @@ cargo make watch-test
 
 **Note:** For more information on publishing AMIs see [publishing](PUBLISHING.md).
 
+### vmware-k8s
+
+First, an initial management cluster needs to be created using [`EKS Anywhere`](https://anywhere.eks.amazonaws.com/docs/getting-started/production-environment/vsphere-getstarted/#create-an-initial-cluster).
+You can then set `TESTSYS_MGMT_CLUSTER_KUBECONFIG` to the path to the management clusters kubeconfig.
+You need to [build](BUILDING.md) Bottlerocket and a publicly accessible [TUF repository](https://github.com/bottlerocket-os/bottlerocket/blob/develop/PUBLISHING.md#repo-location) to test VMware variants.
+Either `Infra.toml` or your environment need to be configured.
+Make sure to set: GOVC_URL, GOVC_USERNAME, GOVC_PASSWORD, GOVC_DATACENTER, GOVC_DATASTORE, GOVC_NETWORK, GOVC_RESOURCE_POOL, GOVC_FOLDER.
```

Comment:
Are these environment variables the only ones that I need to set? In the documentation that you added there isn't any mention about these variables or if it is possible to set them up via `Infra.toml`.

If the `GOVC_` env variables can't be set via `Infra.toml`, make it clearer here, otherwise I suggest you add an snippet of code that shows how to use `Infra.toml` to set said variables.

What else does "your environment" covers? Are there environment variables that can be set for the data centers read from `Infra.toml`? If so, which ones?

Reply:
Those are the only ones needed to be set. The behavior is the same as `pubsys`, and documented in `Infra.toml.example`.

Reply:
> The behavior is the same as pubsys, and documented in Infra.toml.example.

[This section] of the `Infra.toml.example` file covers the fields that will directly map to `GOVC_*`, that's fine. What is confusing from this wording is that it makes me think that I have to set `GOVC_*` as environment variables in my session in order to use testsys with VMware which (AFAIU) isn't actually true, since as you said I can use an existing `Infra.toml` configuration. So probably a bit of clarification will help, since saying `Either `Infra.toml` or your environment need to be configured.` 

```
Make sure you set X, Y, Z either via `Infra.toml` (link to the section that explains GOVC) or with environment variables in your session (if this is supported otherwise omit)
```

[This section]: https://github.com/bottlerocket-os/bottlerocket/blob/8d1ddbb5be6edb6f60ebf34e627403d3ec0108fb/tools/pubsys/Infra.toml.example#L73

Reply:
I have added some clarity.